wrapper.py

```diff
@@ -107,7 +108,17 @@ class PixelInteractive(RodanTask):
     name = 'Pixel_js'
     author = 'Zeyad Saleh, Ke Zhang & Andrew Hankinson'
     description = 'Pixel-level ground truth creation and correction'
-    settings = {'job_queue': 'Python2'}
+    settings = {
+            'title': 'Options',
+            'type': 'object',
+            'properties': {
+                'Output Mask': {
+                    'type': 'boolean',
+                    'default': True
```

Comment:
I still think the default behavior should be generating images instead of masks. The next job after Pixel is usually Calvo trainer, and it expects an `image/rgba+png` resource type.

https://github.com/DDMAL/Calvo-classifier/blob/f38e48390b19f57f9ba8fb9283a1ec14ebd73ec2/fast_calvo_trainer.py#L53-L60

The user may not know in advance if they are going to keep working in the layering session or if that is the latest one, and so it is perhaps not clear when they instantiate the workflow run if they need an image or a mask. Is there any way that we can be choose afterwards?

Reply:
The output is always an `image/rgba+png` resource type. The only difference between mask and image is that in the mask the selection is just a solid color, resulting in a smaller resulting file.

I don't think there's a really good way to choose after that wouldn't involve changing how pixel and rodan communicate in a bigger way. The way workflows are set up is that the inputs and outputs are defined before starting the workflow run and while it could be changed so the output is determined at runtime that would make workflows unable to really be replicated, right?

Reply:
I wasn't thinking about changing the behavior of generating results from *apriori* to *aposteriori*, but just thinking about giving the user more flexibility and streamline the process. 

Indeed the masks are just a solid color, but they are not meaningful for the next job in a workflow (e.g., Calvo) unless it is the same job (i.e., Pixel). 

I guess can be very explicit about this in the documentation and leave with it for the time being. In any case, this behavior is easily changeable.